kitchen-tests/kitchen.dokken.yml

```diff
@@ -180,13 +180,13 @@ platforms:
         - RUN /usr/bin/apt-get update -y
         - RUN /usr/bin/apt-get install ifupdown -y # we need this for /etc/network/interfaces & ifconfig resource testing
 
-  - name: ubuntu-24.04
-    driver:
-      image: dokken/ubuntu-24.04
-      pid_one_command: /bin/systemd
-      intermediate_instructions:
-        - RUN /usr/bin/apt-get update -y
-        - RUN /usr/bin/apt-get install ifupdown -y # we need this for /etc/network/interfaces & ifconfig resource testing
+  # - name: ubuntu-24.04
+  #   driver:
+  #     image: dokken/ubuntu-24.04
+  #     pid_one_command: /bin/systemd
+  #     intermediate_instructions:
+  #       - RUN /usr/bin/apt-get update -y
+  #       - RUN /usr/bin/apt-get install ifupdown -y # we need this for /etc/network/interfaces & ifconfig resource testing
```

Comment:
This is just a configuration item and doesn't need to be commented out. Only the one in the .github workflows file controls what is actually ran .github/workflows/kitchen.yml